src/domain/user.js

```diff
@@ -185,4 +185,17 @@ export default class User {
 
     return foundUsers.map((user) => User.fromDb(user))
   }
+
```

Comment:
This looks fine, but it's perhaps a bit inconsitent with the other methods on the user object - rather than have this as a static method, could you have your controller load the user record find with `User.findById`, update the cohort Id property of that user, and then just call `update` on the user instance? That way, your contorller can also handle the case when an invalid user is supplied (since findUser will return null I think), and it's a bit more consistent and doesn't require adding any new methods to user?